roles/openshift_hosted_facts/tasks/main.yml

```diff
@@ -1,4 +1,10 @@
 ---
+- name: Set legacy hosted facts
+  openshift_facts:
+    role: hosted
+    openshift_env:
+      openshift_hosted_router_selector: "{{ openshift_router_selector | default(None) }}"
```

Comment:
yeah, I'd just like to see us limit the number of calls into openshift_facts at some point... Is the oo_openshift_env filter a better place to try to tackle this?